exporter/stackdriver/stackdriver.go

```diff
@@ -73,9 +80,34 @@ type Exporter struct {
 	statsExporter *statsExporter
 }
 
+// Register sets up both stats and tracing exporting.
+func Register(o Options) (done func(), err error) {
```

Comment:
Seems more like a convenience method that a user should make on their own than something we want to expose in our API, could we perhaps make it a helper for the test as you had done?

Reply:
My goal here was to make it as simple as possible for people to "turn on" OpenCensus for GCP client libraries. Adding this method makes this a one-liner as opposed to a 6-liner. @rakyll what do you think?

Reply:
I agree that done might be giving the wrong impression that it should be deferred right away. What about calling it unregister and document that it is more likely never be called but can be called to unregister and flush?